README.md

```diff
@@ -306,3 +306,28 @@ sender.Close()
 
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## Releasing this SDK
```

Comment:
Please add this section to the Table of Contents at the top of the README.

Reply:
Done.